.buildkite/bk.integration.pipeline.yml

```diff
@@ -28,6 +28,22 @@ steps:
       image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci-with-hooks:0.5"
       useCustomGlobalHooks: true
 
+#   FIXME: implement once an ECH FRH environment is available
+#  - label: Start ECH FRH stack for FIPS integration tests
+#    key: integration-frh
+#    env:
+#      ASDF_TERRAFORM_VERSION: 1.9.2
+#    command: |
+#      #!/usr/bin/env bash
+#      set -euo pipefail
+#      source .buildkite/scripts/steps/frh_start.sh
+#    artifact_paths:
+#      - test_infra/frh/*.tfstate
```

Comment:
I'm not a big fan of archiving the terraform state, it might contain secrets in plain.

Reply:
Sure, I just copied this block from another part of the BK pipeline.  We can omit it from both places for consistency but I would prefer it if we omitted it from the other place in a separate PR first as this concern is unrelated to the topic of this PR here.